src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -17,7 +17,7 @@ import {isContainUrl} from '../../../utils/Utils';
 import {requestExternalStoragePermission} from '../../../utils/permission';
 import {setChannel} from '../../../context/actions/setChannel';
 import {setParticipants} from '../../../context/actions/groupChat';
-import {uploadFile} from '../../../service/file';
+import ImageUtils from '../../../utils/image';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been changed from `uploadFile` to `ImageUtils`. Ensure that the `ImageUtils` module contains the necessary functionality for image uploading and that it is correctly implemented.



<!-- This is an auto-generated comment by OSS CodeRabbit -->